test/unistd/access.c

```diff
@@ -65,6 +66,57 @@ int main() {
   printf("F_OK(%s): %d\n", "renamedfile", faccessat(AT_FDCWD, "renamedfile", F_OK, 0));
   printf("errno: %d\n", errno);
 
+  EM_ASM(
+    var fchmodstream = FS.open("fchmodtest", "r");
+#if WASMFS
+    FS.fchmod(fchmodstream, 0o000);
+#else
+    FS.fchmod(fchmodstream.fd, 0o000);
+#endif
+  );
+  struct stat fileStats;
+  stat("fchmodtest", &fileStats);
+  assert(!(fileStats.st_mode & 0777));
+
+#if !NODE*FS
+  EM_ASM(
+    FS.symlink('forbidden', 'symlinkfile');
+    FS.lchmod('symlinkfile', 0o777);
+  );
+  
+  struct stat symlinkStats;
+
+  lstat("symlinkfile", &symlinkStats);
+  assert(symlinkStats.st_mode & 0777);
```

Comment:
I'm not sure which is better of `0777` and `0o777`, but let's pick one and be consistent?

If nothing else, `0777` (without `o`) is shorter which might be better.

Reply:
I think that JS allows both `0777` and `0o777`, but C only allows `0777`, so I think it makes sense to switch them all to `0777` since I use some of the values in C assert statements.

Reply:
Ah, good point, I'd forgotten about that... yes, `0777` sounds best then overall.